### PR TITLE
Login form

### PR DESCRIPTION
**Pull request by Shinyn**
Added login form and register form

### CODE REVIEW

File: instapups/src/Pages/Login.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 
 const Login = () => {
   return (
-    <div>Login</div>
-  )
-}
+    <div>
+      <input type="text" placeholder="Username" className="login-username" />
+      <input type="password" placeholder="Password" className="login-password" />
+      <button className="login-button" onClick={goToProfile}>
```
Comment (Darkshiira) [inline instapups/src/Pages/Login.js:8]: Instead of an onclick, add an onsubmit on form

File: instapups/src/Pages/Login.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react";
 
 const Login = () => {
   return (
-    <div>Login</div>
-  )
-}
+    <div>
```
Comment (Darkshiira) [inline instapups/src/Pages/Login.js:5]: Add a form

File: instapups/src/Pages/Login.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react";
```
Comment (Darkshiira) [inline instapups/src/Pages/Login.js:1]: Import useState and useEffect

File: instapups/src/Pages/Login.js [hunk 4]
```diff
@@ -1,9 +1,20 @@
...
 
 const Login = () => {
   return (
-    <div>Login</div>
-  )
-}
+    <div>
+      <input type="text" placeholder="Username" className="login-username" />
```
Comment (Darkshiira) [inline instapups/src/Pages/Login.js:6]: Save the value to a usestate onChange

File: instapups/src/Pages/Login.js [hunk 5]
```diff
@@ -1,9 +1,20 @@
...
+      <input type="password" placeholder="Password" className="login-password" />
```
Comment (Darkshiira) [inline instapups/src/Pages/Login.js:7]: Save the value to a useState onChange

File: instapups/src/Pages/Register.js [hunk 6]
```diff
@@ -1,9 +1,22 @@
...
 const Register = () => {
   return (
-    <div>Register</div>
-  )
-}
+    <div>
+      <input type="text" placeholder="Username" className="register-username" />
+      <input type="password" placeholder="Password" className="register-password" />
+      <input type="password" placeholder="Repeat password" className="register-repeat-password" />
+      <button className="register-button" onClick={goToLogin}>
```
Comment (Darkshiira) [inline instapups/src/Pages/Register.js:9]: Add an onSubmit on form instead

File: instapups/src/Pages/Register.js [hunk 7]
```diff
@@ -1,9 +1,22 @@
-import React from 'react'
+import React from "react";
```
Comment (Darkshiira) [inline instapups/src/Pages/Register.js:1]: Import useState and useEffect

File: instapups/src/Pages/Register.js [hunk 8]
```diff
@@ -1,9 +1,22 @@
...
 
 const Register = () => {
   return (
-    <div>Register</div>
-  )
-}
+    <div>
```
Comment (Darkshiira) [inline instapups/src/Pages/Register.js:5]: Add a form

File: instapups/src/Pages/Register.js [hunk 9]
```diff
@@ -1,9 +1,22 @@
...
+      <input type="text" placeholder="Username" className="register-username" />
```
Comment (Darkshiira) [inline instapups/src/Pages/Register.js:6]: Save value in a state onchange

File: instapups/src/Pages/Register.js [hunk 10]
```diff
@@ -1,9 +1,22 @@
...
+      <input type="password" placeholder="Password" className="register-password" />
```
Comment (Darkshiira) [inline instapups/src/Pages/Register.js:7]: Save value onchange to a state

**Review by Darkshiira — COMMENTED** [review]
Theese are the changes I would do if I wrote the code, because the values needs to be saved somewhere to be sent in the fetch. And the entire form needs to be submitted instead of an onclick-function

**Comment by Shinyn** [comment]
Will apply changes / I'll Be Back

**Pull request closed without merging**